### DISCUSSION TITLE
Welcome to GarminDB Discussions!

### DISCUSSION THREAD

**Discussion by tcgoetz** [tcgoetz/GarminDB#93] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by irvingbennett** [comment]
Hello tcgoetz! Great project! Downloaded your code, ran it on Windows 11 in the Anaconda environment, and it worked great. Thanks for sharing.

**Comment by KS1959** [comment]
Hi, complete (tech) novice.  But I would be interested in whether the daily data, training load, endurance score etc is accessible?  Has anyone tried to access that?
